cla_backend/libs/eligibility_calculator/tests/cfe_civil/test_savings.py

```diff
@@ -21,21 +21,13 @@ def test_bank_balance_and_investments(self):
                     "subject_matter_of_dispute": False
                 },
             ],
-            "non_liquid_capital": [],
-        }}
-        self.assertEqual(expected, output)
-
-    def test_assets(self):
-        savings = Savings(bank_balance=0, investment_balance=0, asset_balance=80011)
-        output = translate_savings(savings)
-        expected = {"capitals": {
-            "bank_accounts": [],
             "non_liquid_capital": [
                 {
-                    "value": 800.11,
+                    "value": 2,
                     "description": "Asset",
                     "subject_matter_of_dispute": False
                 },
-            ]
+
+            ],
         }}
```

Comment:
Can we have a test to see what happens when Savings is empty? i.e. to show this fixes the [empty savings bug]](https://mojdt.slack.com/archives/C04AW468PU4/p1702658658739109):

> On the /about page you say you have savings, and on pressing "Continue" it complains PropertyExpectedException: 'Savings' requires attribute 'bank_balance' and was not given at __init__

i.e.
```
    def test_empty_savings(self):
        savings = Savings()

        output = translate_savings(savings)

        expected = {}
        self.assertEqual(expected, output)
```